mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go

```diff
@@ -0,0 +1,124 @@
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"google.golang.org/api/googleapi"
+)
+
+func TestAccSecurityCenterOrganizationMuteConfig_createUpdateDelete(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+		"location":      "global",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck: func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: map[string]func() (*schema.Provider, error){
+
+			"google": google.Provider,
+		},
+		CheckDestroy: testAccCheckSecurityCenterOrganizationMuteConfigDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterOrganizationMuteConfig(context),
+			},
+			{
+				ResourceName:            "google_scc_v2_organization_mute_config.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"organizationmute_ConfigId", "parent"},
+			},
+			{
+				Config: testAccSecurityCenterOrganizationMuteConfig_update(context),
+			},
+			{
+				ResourceName:            "google_scc_v2_organization_mute_config.primary",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"organizationmute_ConfigId", "parent"},
+			},
+			{
+				Config: testAccSecurityCenterOrganizationMuteConfig_delete(context),
+			},
+		},
+	})
+}
+
+func testAccSecurityCenterOrganizationMuteConfig(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_scc_v2_organization_mute_config" "primary" {
+  mute_config_id = "tf-test-my-config-%{random_suffix}"
+  parent         = "organizations/%{org_id}"
+  filter         = "category: \"OS_VULNERABILITY\""
+  location       = "%{location}"
+  description    = "My Mute Config"
+  type           = "STATIC"
+}
+`, context)
+}
+
+func testAccSecurityCenterOrganizationMuteConfig_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_scc_v2_organization_mute_config" "primary" {
+  mute_config_id = "tf-test-my-config-%{random_suffix}"
+  parent         = "organizations/%{org_id}"
+  filter         = "category: \"WEB_SECURITY\""
+  location       = "%{location}"
+  description    = "My Mute Config Updated"
+  type           = "STATIC"
+}
+`, context)
+}
+
+func testAccSecurityCenterOrganizationMuteConfig_delete(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_scc_v2_organization_mute_config" "primary" {
+  mute_config_id = "tf-test-my-config-%{random_suffix}"
+  parent         = "organizations/%{org_id}"
+  filter         = "category: \"OS_VULNERABILITY\""
+  location       = "%{location}"
+  description    = "My Mute Config"
+  type           = "STATIC"
+}
+`, context)
+}
+
+func testAccCheckSecurityCenterOrganizationMuteConfigDestroyProducer(t *testing.T) resource.TestCheckFunc {
```

Comment:
The generated test file will contain a DestroyProducer. Check google-beta/services/securitycenterv2/resource_scc_v2_organization_mute_config_generated_test.go i.e. https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10989-old..auto-pr-10989#diff-5b7cf01d02c77b0af605b1c63bd222a8e7283b56cdf289e6fbbbb878a21e07ef

Reply:
addressed